hack/release.sh

```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```

Comment:
Ugh, this is actually because `something=` doesn't do globs like it should.  If you change this line to the following, it should work:
```bash
DEBFILE=( bundles/$VERSION/ubuntu/lxc-docker*.deb )
```

(and then `$DEBFILE` will get the first one, just like `$BASH_SOURCE` gets the first item in the `BASH_SOURCE` array)